lib/12T/ext4mag.py

```diff
@@ -0,0 +1,145 @@
+# ext4mag.py
+# Author:  Landon Burleson/James Stine
+# Organization: Oklahoma State University
+# E-mail: landon.burleson@okstate.edu
+# Description: extract info from magic at terminal
```

Comment:
is that different from the other ext2mag file in the repo?

Reply:
again, not really needed.  Deleted.